addons/sourcemod/scripting/vip/adminmenu/List.sp

```diff
@@ -12,9 +12,13 @@ void ShowVipPlayersListMenu(int iClient)
 	hMenu.SetTitle("%T:\n ", "MENU_LIST_VIP", iClient);
 	hMenu.ExitBackButton = true;
 
-	hMenu.AddItem("search", "Найти игрока\n ");
-
-	hMenu.AddItem("show_all", "Показать всех\n ");
+	char szName[32];
```

Comment:
do not do this. Such a variable already exists, you can simply use it
![image](https://user-images.githubusercontent.com/5843406/128836879-a9751f13-d1b6-4e9d-897d-0a88ed5c56f9.png)